templates/patterns/modal/pattern-modal.html.twig

```diff
@@ -11,7 +11,7 @@
 {% block trigger %}
   {{ pattern('button', {
     label: btn_label,
-    variant: btn_style,
+    style: btn_style,
```

Comment:
Let's be verbose, "button_variant".

Reply:
Also note that we are generating the id in the template, which is wrong.

Reply:
Actually let's move everything to a "button:" entry so we can pass all the button pattern info in there.